src/bicgstab.cpp

```diff
@@ -119,7 +119,7 @@ int bicgstabL(const int L, const int n, realnum *x,
 
```

Comment:
The returned value is not just an error code but could also be the required workspace. As the type of `n` is meep::integer, the return type must also be meep::integer.